templates/al2023/provisioners/install-nvidia-driver.sh

```diff
@@ -12,7 +12,14 @@ echo "Installing NVIDIA ${NVIDIA_DRIVER_MAJOR_VERSION} drivers..."
 ################################################################################
 ### Add repository #############################################################
 ################################################################################
-sudo dnf config-manager --add-repo https://developer.download.nvidia.com/compute/cuda/repos/amzn2023/x86_64/cuda-amzn2023.repo
+# Determine the domain based on the region
+if [ "$AWS_REGION" == "cn-*" ]; then
```

Comment:
Does this work? In testing I had to use:

```
if [[ $AWS_REGION == cn-* ]]; then
```

Reply:
oh, you are right, updated!